doc/en/user/source/extensions/css/workshop/point.rst

```diff
@@ -0,0 +1,708 @@
+   
+      [10 - LABELRANK]
+   
+   This expression will result in values between 0 and 10 and will be used for the **-gt-label-priority**.
+
+   .. code-block:: css
+      :emphasize-lines: 2,9
+
+      * {
+        mark-size: [10-(SCALERANK/2)];
+
+        font-fill: black;
+        font-family: "Arial";
+        font-size: 10;
+
+        label-anchor: 0.5 0;
+        label-offset: 0 [10-(SCALERANK/2)];
+
+        -gt-mark-label-obstacle: true;
+        -gt-label-max-displacement: 90;
+        -gt-label-padding: 2;
+        -gt-label-priority: [10 - LABELRANK];
+      }
+   
+   .. image:: img/point_07_expression.png
+
+#. Next we can use ``FEATURECLA`` to check for capital cities.
+
+   Adding a selector for capital cities at the top of the file:
+
+   .. code-block:: css
+
+      /* capitals */
+      [@scale < 70000000]
+      [FEATURECLA = 'Admin-0 capital']  {
+         mark: symbol(star);
+         label: [NAME];
+      }
+      [@scale > 70000000] [SCALERANK < 2]
+      [FEATURECLA = 'Admin-0 capital']  {
+         mark: symbol(star);
+         label: [NAME];
+      }
+   
+   And updating the populated places selectors to ignore capital cities:
+
+   .. code-block:: css
+
+      /* populated places */
+      [@scale < 4000000]
+      [FEATURECLA <> 'Admin-0 capital'] {
+         mark: symbol(circle);
+         label: [NAME];
+      }
+      [@scale > 4000000] [@scale < 8000000] [SCALERANK < 7]
+      [FEATURECLA <> 'Admin-0 capital'] {
+         mark: symbol(circle);
+         label: [NAME];
+      }
+
+      [@scale > 8000000] [@scale < 17000000] [SCALERANK < 5]
+      [FEATURECLA <> 'Admin-0 capital'] {
+         mark: symbol(circle);
+         label: [NAME];
+      }
+
+      [@scale > 17000000] [@scale < 35000000] [SCALERANK < 4]
+      [FEATURECLA <> 'Admin-0 capital'] {
+         mark: symbol(circle);
+         label: [NAME];
+      }
+
+      [@scale > 35000000] [@scale < 70000000][SCALERANK < 3]
+      [FEATURECLA <> 'Admin-0 capital'] {
+         mark: symbol(circle);
+         label: [NAME];
+      }
+
+      [@scale > 70000000] [@scale < 140000000][SCALERANK < 2]
+      [FEATURECLA <> 'Admin-0 capital'] {
+         mark: symbol(circle);
+         label: [NAME];
+      }
+
+      [@scale > 140000000] [SCALERANK < 1]
+      [FEATURECLA <> 'Admin-0 capital'] {
+         mark: symbol(circle);
+         label: [NAME];
+      }
+
+   .. image:: img/point_08_symbol.png
+
+#. Finally we can fill in the capital city symbols using a combination of a selector to detect capital cities, and pseudo selector to provide mark styling.
+
+   .. code-block:: css
+   
+      [FEATURECLA = 'Admin-0 capital'] :mark {
+        fill: black;
+      }
+
+      :symbol {
+        fill: gray;
+        stroke: black;
+      }
+
+   .. image:: img/point_09_fill.png
+
+#. If you would like to check your work the final file is here: :download:`point_example.css </files/point_example.css>`
+
+
+Additional Considerations
+-------------------------
```

Comment:
Hmmmm, wait you want me to separate the entire section? I can do that, but not quite sure how... I'll ping you on this one.